rust/parquet/Cargo.toml

```diff
@@ -40,6 +40,7 @@ zstd = { version = "0.5", optional = true }
 chrono = "0.4"
 num-bigint = "0.3"
 arrow = { path = "../arrow", version = "2.0.0-SNAPSHOT", optional = true }
+base64 = { version = "*", optional = true }
```

Comment:
Not using a specific version as we already transitively depend on an older version, but I'll change this when the feature changes are merged